cpu_ticks_static.patch

From: Frediano Ziglio <frediano.ziglio@citrix.com>

Make cpu_enable_ticks and cpu_disable_ticks static.
These function are defined and used just in vl.c.

diff --git a/sysemu.h b/sysemu.h
index 45eeb10..eb2accd 100644
--- a/sysemu.h
+++ b/sysemu.h
@@ -22,8 +22,6 @@ void vm_start(void);
 void vm_stop(int reason);
 
 int64_t cpu_get_ticks(void);
-void cpu_enable_ticks(void);
-void cpu_disable_ticks(void);
 
 void qemu_system_reset_request(void);
 void qemu_system_shutdown_request(void);
diff --git a/vl.c b/vl.c
index fb1c199..2a5dbc4 100644
--- a/vl.c
+++ b/vl.c
@@ -961,7 +961,7 @@ static int64_t cpu_get_clock(void)
 }
 
 /* enable cpu_get_ticks() */
-void cpu_enable_ticks(void)
+static void cpu_enable_ticks(void)
 {
     if (!cpu_ticks_enabled) {
         cpu_ticks_offset -= cpu_get_real_ticks();
@@ -972,7 +972,7 @@ void cpu_enable_ticks(void)
 
 /* disable cpu_get_ticks() : the clock is stopped. You must not call
    cpu_get_ticks() after that.  */
-void cpu_disable_ticks(void)
+static void cpu_disable_ticks(void)
 {
     if (cpu_ticks_enabled) {
         cpu_ticks_offset = cpu_get_ticks();
